community/contributing/index.md

```diff
@@ -8,22 +8,34 @@ redirect_from: /docs/contributing-to-spinnaker
 
 {% include toc %}
 
-## Filing issues
+Welcome to the Spinnaker community! Please join us as a contributor.
 
-Please file issues through the [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues).
+## Quick Links
 
-## How to Become a Contributor
+- [Spinnaker GitHub issue tracker](https://github.com/spinnaker/spinnaker/issues): file Spinnaker issues
```

Comment:
I think I'm in favor of having something like "File an issue" as the heading, and "Spinnaker issue tracker" as the link text.

Reply:
good suggestion